### PR TITLE
Fixed backwards compat with older versions of Grafana [Prometheus]

### PR DESCRIPTION
**Pull request by Arkatufus**
close https://github.com/petabridge/phobos-dashboards/issues/20

### CODE REVIEW

File: prometheus/phobos-2.x/phobos-akkanet-cluster-overview-OpenTelemetry.json [hunk 1]
```diff
@@ -1054,10 +1046,9 @@
         {
           "datasource": {
             "type": "prometheus",
-            "uid": "${DS_PROMETHEUS}"
+            "uid": "P1809F7CD0C75ACF3"
```
Comment (Aaronontheweb) [inline prometheus/phobos-2.x/phobos-akkanet-cluster-overview-OpenTelemetry.json:1049]: I think this is wrong - shouldn't this value be populated via a DS_PROMETHEUS variable?

Comment (Arkatufus) [reply prometheus/phobos-2.x/phobos-akkanet-cluster-overview-OpenTelemetry.json:1049]: I don't know, the input portion of the json file got deleted in the export.
and there are actually 2 `pluginVersion` in the exported files

Comment (Arkatufus) [reply prometheus/phobos-2.x/phobos-akkanet-cluster-overview-OpenTelemetry.json:1049]: I think the export function is broken somehow

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Pull request merged**